test/gomod.collector.test.ts

```diff
@@ -9,7 +9,7 @@ describe('Golang go.mod parser test', () => {
   const collector: GomodDependencyCollector = new GomodDependencyCollector(fakeSourceRoot);
 
   it('tests valid go.mod', async () => {
-    fake(`go list -f '{{ join .Imports "\\n" }}' ./...`, `github.com/alecthomas/units
+    fake(`go list -mod=readonly -f '{{ join .Imports "\\n" }}' ./...`, `github.com/alecthomas/units
```

Comment:
This changes seems to be too intrusive. Can this be avoided?

Reply:
Any suggestions to avoid / loosen this kind of test dependency on actual command?

Reply:
Moved command to common place using utils function, PTAL

Reply:
Moved command to common place using utils function, PTAL